CNV#37545a: RN: enhancement Specializing a Windows VM flow during VM creation (#78380)
openshift/openshift-docs: virt/release_notes/virt-4-16-release-notes.adoc

//CNV-34687 Release note: Generally available (GA) CPU Hotplug
* Hot plugging virtual machines is now generally available. If a virtual machine cannot be hot plugged, the condition `RestartRequired` is applied to the VM. You can view this condition in the *Diagnostics* tab of the web console.

//CNV-37545 RN: enhancement Specializing a Windows VM flow during VM creation

> * You can now select `sysprep` options when you create a Windows VM from an instance type. Previously it was necessary to set the `sysprep` options by customizing the VM after its creation.

Reviewer: ```suggestion
* You can now select `sysprep` options when you create a MicroSoft Windows virtyal machine (VM) from an instance type. Previously, you had to set the `sysprep` options by customizing the VM after its creation.
```

Author: Because this is release note, VM has already been clarified as virtual machine earlier in the assembly.

* VMs require a minimum of 1 GiB of allocated memory to enable memory hotplug. If a VM has less than 1 GiB of allocated memory, then memory hotplug is disabled.

//CNV-34681: runbooks moved to openshift/runbooks repo

> * Runbooks for {VirtProductName} alerts are now maintained only in the https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[`openshift/runbooks` git repository] and have been removed from the documentation.

Reviewer: ```suggestion
* Runbooks for {VirtProductName} alerts are now only maintained in the https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[`openshift/runbooks` git repository] and have been removed from the documentation.
```

![Screenshot from 2024-07-03 16-29-12](https://github.com/openshift/openshift-docs/assets/57954076/d409c804-ea51-46dc-a48b-a60661d1e7c5)

Author: I did not write or update the note about runbooks, so I will leave as is.

Author: But I believe that the placement of 'only' in this sentence is correct--the alerts are maintained only in the location specified.

Reviewer: Thanks for clarifying.
